dev-data/aws-badge.data.ts

```
@@ -0,0 +1,20 @@
+import { COURSE_TITLES } from './course-titles.data';
+import cloudBadge from '@/shared/assets/aws-cloud-pract-badge.webp';
+import aiBadge from '@/shared/assets/aws-ai-practitioner-badge.webp';
```

ESLint

[error] `@/shared/assets/aws-ai-practitioner-badge.webp` import should occur before import of `@/shared/assets/aws-cloud-pract-badge.webp`

(import/order)